Resources/doc/events.rst

```diff
@@ -50,54 +50,54 @@ Create the Event object
 The event object allows to pass some data to the listener. In this case,
 it will hold the menu being created and the factory.
 
-.. code-block:: yaml
-
-  <?php
-  // src/Acme/DemoBundle/Event/ConfigureMenuEvent.php
-
-  namespace Acme\DemoBundle\Event;
-
-  use Knp\Menu\FactoryInterface;
-  use Knp\Menu\ItemInterface;
-  use Symfony\Component\EventDispatcher\Event;
-
-  class ConfigureMenuEvent extends Event
-  {
-      const CONFIGURE = 'acme_demo.menu_configure';
-
-      private $factory;
-      private $menu;
-
-      /**
-       * @param \Knp\Menu\FactoryInterface $factory
-       * @param \Knp\Menu\ItemInterface $menu
-       */
-      public function __construct(FactoryInterface $factory, ItemInterface $menu)
-      {
-          $this->factory = $factory;
-          $this->menu = $menu;
-      }
-
-      /**
-       * @return \Knp\Menu\FactoryInterface
-       */
-      public function getFactory()
-      {
-          return $this->factory;
-      }
+.. code-block:: php
 
-      /**
-       * @return \Knp\Menu\ItemInterface
-       */
-      public function getMenu()
-      {
-          return $this->menu;
-      }
-  }
+    <?php
+    // src/Acme/DemoBundle/Event/ConfigureMenuEvent.php
+
+    namespace Acme\DemoBundle\Event;
```

Comment:
Good idea. I've just updated all the docs to use the AppBundle where appropriate. Thanks.